mvn-defaults/pom.xml

```diff
@@ -484,7 +487,118 @@
                 <version>1.7.1</version>
             </extension>
         </extensions>
+
         <plugins>
+
```

Comment:
commented out here.

Reply:
I'll remove this to tidy. If we want the code back to build on, it's in our git log.